java/core/src/main/java/com/whylogs/core/ColumnProfile.java

```diff
@@ -0,0 +1,80 @@
+package com.whylogs.core;
+
+import com.whylogs.core.metrics.Metric;
+import com.whylogs.core.metrics.OperationResult;
+import com.whylogs.core.schemas.ColumnSchema;
+import com.whylogs.core.views.ColumnProfileView;
+import java.util.ArrayList;
+import java.util.HashMap;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class ColumnProfile<T> {
+  // Required
+  private String name;
+  private ColumnSchema schema;
+  private int cachedSize;
```

Comment:
s/cachedSize/cacheSize/

Reply:
Sorry, I don't know what you're wanting here? Could you clarify your comment?

Reply:
Is this trying to say to change it to cachSize? This comment isn't very clear to me.

Reply:
Sorry, I sometimes use sed notation. Yes, I'm saying change `cachedSize` to `cacheSize`.